pkg/asset/ignition/install_ignition.go

```diff
@@ -70,27 +71,32 @@ func (i *InstallIgnition) Generate(dependencies asset.Parents) error {
 		},
 	}
 
-	// Add user 'core' password if required
 	if applianceConfig.Config.UserCorePass != nil {
+		// Avoid machine reboot after bootstrap to debug install ignition
```

Comment:
Does this comment belong here? it is rather confusing

Reply:
Nope:) fixed.

Reply:
Looks great now. Approved.